### ISSUE TITLE
Mishards loses connection to the rw node. 

### ISSUE THREAD

**Issue by Di-Weng** [milvus-io/milvus#4567]
**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**

**Steps/Code to reproduce behavior**
Follow this [guide](http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) to craft a minimal bug report. This helps us reproduce the issue you're having and resolve the issue more quickly.
When successively insert 4096 x 2048 vectors every 8 seconds. Mishards would report that it loses the connection to the rw nodes. It would usually happen after inserts 1000w 2048-dimension vectors.
 
**Expected behavior**

**Method of installation**
- [x] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Softwars conditions (OS, CPU, GPU, Memory)
There are 4 machines in total.
1. Ubuntu 18.04; 2 x Intel(R) Xeon(R) CPU E5-2678 v3 @ 2.50GHz; 4 x TITAN RTX; 128GB memory
    - mysql (meta data)
2. CentOS Linux release 7.6.1810;  Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz; no GPU; 128GB memory
    - ip: 192.168.50.21
    - rw node(docker ip: 172.21.0.2)
    - mishards service
3. CentOS Linux release 7.6.1810;  Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz; no GPU; 128GB memory
    - ro node
4. CentOS Linux release 7.6.1810;  Intel(R) Xeon(R) CPU E5-2650 v4 @ 2.20GHz; no GPU; 128GB memory
    - ro node

- Milvus version (master or released version)
Milvus: 0.10.4
pymilvus: 0.2.14

**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

ro_server
```yaml
version: 0.5
cluster:
  enable: true
  role: ro
general:
  timezone: UTC+8
  meta_uri: mysql://account:password@192.168.50.149:3309/milvus
network:
  bind.address: 0.0.0.0
  bind.port: 19560
  http.enable: true
  http.port: 19151
storage:
  path: /var/lib/milvus
  auto_flush_interval: 15
cache:
  cache_size: 45GB
  insert_buffer_size: 5GB
wal:
  enable: true
  recovery_error_ignore: false
  buffer_size: 256MB
  path: /var/lib/milvus/wal
gpu:
  enable: false
  cache_size: 1GB
  gpu_search_threshold: 1000
  search_devices:
    - gpu0
  build_index_devices:
    - gpu0
fpga:
   enable: false
   search_devices:
     - fpga0
logs:
  level: debug
  trace.enable: true
  path: /var/lib/milvus/logs
  max_log_file_size: 1024MB
  log_rotate_num: 0
metric:
  enable: false
  address: 127.0.0.1
  port: 9091
```

rw_server
```yaml
version: 0.5
cluster:
  enable: true
  role: rw
general:
  timezone: UTC+8
  meta_uri: mysql://account:password@192.168.50.149:3309/milvus
network:
  bind.address: 0.0.0.0
  bind.port: 19560
  http.enable: true
  http.port: 19151
storage:
  path: /var/lib/milvus
  auto_flush_interval: 15
cache:
  cache_size: 45GB
  insert_buffer_size: 5GB
wal:
  enable: true
  recovery_error_ignore: false
  buffer_size: 256MB
  path: /var/lib/milvus/wal
gpu:
  enable: false
  cache_size: 1GB
  gpu_search_threshold: 1000
  search_devices:
    - gpu0
  build_index_devices:
    - gpu0
fpga:
   enable: false
   search_devices:
     - fpga0
logs:
  level: debug
  trace.enable: true
  path: /var/lib/milvus/logs
  max_log_file_size: 1024MB
  log_rotate_num: 0
metric:
  enable: false
  address: 127.0.0.1
  port: 9091
```

**Screenshots**
![image](https://user-images.githubusercontent.com/37477885/103602750-c87f7280-4f47-11eb-9944-097e78c75076.png)
![image](https://user-images.githubusercontent.com/37477885/103602838-ff558880-4f47-11eb-92a9-bede7fd02110.png)

**Additional context**

**Comment by BossZou** [comment]
@Di-Weng , may you provide the mishards version and config information?

**Comment by Di-Weng** [comment]
mishards version(from docker): milvusdb/mishards:latest; image id:a2ac48d94eaf
config:
```yaml
version: "2.3"
services:
    milvus_wr:
        restart: always
        image: milvusdb/milvus:0.10.4-cpu-d120220-e72454
        volumes:
            - /data/wengdi/milvus_dist/db:/var/lib/milvus/db
            - ./wr_server.yml:/var/lib/milvus/conf/server_config.yaml
            - /data/wengdi/milvus_dist/logs:/var/lib/milvus/logs

    jaeger:
        restart: always
        image: jaegertracing/all-in-one:1.14
        ports:
            - "0.0.0.0:5775:5775/udp"
            - "0.0.0.0:16686:16686"
            - "0.0.0.0:9441:9441"
        environment:
            COLLECTOR_ZIPKIN_HTTP_PORT: 9411

    mishards:
        restart: always
        image: milvusdb/mishards
        ports:
            - "0.0.0.0:19531:19531"
            - "0.0.0.0:19532:19532"
        volumes:
            - /data/wengdi/milvus_dist/db:/tmp/milvus/db
        command: ["python", "mishards/main.py"]
        environment:
            FROM_EXAMPLE: 'true'
            SQLALCHEMY_DATABASE_URI: mysql+pymysql://account:password@192.168.50.149:3309/milvus?charset=utf8mb4
            DEBUG: 'true'
            SERVER_PORT: 19531
            WOSERVER: tcp://milvus_wr:19560
            DISCOVERY_PLUGIN_PATH: static
            DISCOVERY_STATIC_HOSTS: milvus_wr, 192.168.50.22, 192.168.50.23
            DISCOVERY_STATIC_PORT: 19560
            TRACER_CLASS_NAME: jaeger
            TRACING_SERVICE_NAME: mishards-demo
            TRACING_REPORTING_HOST: jaeger
            TRACING_REPORTING_PORT: 5775
            LOG_LEVEL: DEBUG
            LOG_PATH: /data/wengdi/milvus_dist/mishards_log
            LOG_NAME: mishards.log
        depends_on:
            - milvus_wr
            - jaeger

```

**Comment by BossZou** [comment]
@Di-Weng thanks. May you provide the mishards output logs here? If you deploy with docker image, the log could be output on docker consul, you can run `docker logs {container-id}` to get it.

**Comment by Di-Weng** [comment]
> @Di-Weng thanks. May you provide the mishards output logs here? If you deploy with docker image, the log could be output on docker consul, you can run `docker logs {container-id}` to get it.

The first comment has included the screenshots of mishards output logs.

**Comment by BossZou** [comment]
@Di-Weng , I see. Did you restart your write node ? Which deploy method did you use, with k8s or with static-method ?

**Comment by Di-Weng** [comment]
> @Di-Weng , I see. Did you restart your write node ? Which deploy method did you use, with k8s or with static-method ?

The write node is always working online. And I use the static-method.

**Comment by Di-Weng** [comment]
Well, after restarting the services of all nodes, the whole system is working fine. I think this bug is an incidence which could only happen when importing too many data import mishards in a very short period of time.

**Comment by BossZou** [comment]
@Di-Weng , we'll try to reproduce. Did you specify timeout when inserting?

**Comment by Di-Weng** [comment]
> @Di-Weng , we'll try to reproduce. Did you specify timeout when inserting?

Insert 17360*2048 vectors with float as its type every 8 seconds, following with a 2-second pause. I use thread pool to insert the vectors in asynchronous way. I manually set the size of  thread pool to 30 (In my previous experiments, too many connection at the same time would make the system not respond :) ).

**Comment by BossZou** [comment]
@Di-Weng , I might reproduced in my host machine. My write node has crashed when client received error. So you may check 
 if your write node service is available. We'll fix this issue at next version.

**Comment by BossZou** [comment]
@Di-Weng , I found the reason the write node crashed in my host machine is that the local disk is full. Anyway, it seem like that there is some problem on write node, cause the write node stop responding request. So you may check the logs of write node and provide here?

**Comment by Di-Weng** [comment]
> @Di-Weng , I found the reason the write node crashed in my host machine is that the local disk is full. Anyway, it seem like that there is some problem on write node, cause the write node stop responding request. So you may check the logs of write node and provide here?

I could not find the original logs of the write node, but I suppose the initial comment includes a screenshot of the docker logs of the write node.

**Comment by BossZou** [comment]
@Di-Weng, I see. Actually Milvus output the logs to log file. I noticed the log path in your config file is `var/lib/milvus/logs`, if you didn't map the log directory to your host machine, you may log into write node container and find the log files.

**Comment by Di-Weng** [comment]
> @Di-Weng, I see. Actually Milvus output the logs to log file. I noticed the log path in your config file is `var/lib/milvus/logs`, if you didn't map the log directory to your host machine, you may log into write node container and find the log files.

Well, the container has been replaced by the following experiments :(

**Comment by BossZou** [comment]
@Di-Weng . All right. You are recommended to reserve log files and once some error occur, them are very helpful to locate issue.

**Comment by yhmo** [comment]
This may caused by a random crash issue #5398. We have fixed it in v1.1.1
Inactive and close.
